containerized_ml_models/model_1/api.py

```diff
@@ -0,0 +1,30 @@
+from flask import Flask, jsonify, request
+import pickle
+import numpy as np
+import json
+
+app = Flask(__name__)
+
+
+@app.route("/ping")
+def ping_success():
+    return jsonify({"ping": "success"})
+
+
+@app.route("/predict", methods=["POST"])
+def predict():
+    record = request.get_data(as_text=True)
+    data = np.array(json.loads(record))
+    if len(data.shape) != 2 or data.shape[1] != 10:
+        raise ValueError(
+            "Illegal Argument of incorrect shape. "
+            "Input should be a list of shape (n,10)."
+        )
```

Comment:
To i tak nie ma znaczenia dla działania api, bo jeśli strzelisz do api requestem, który nie ma odpowiedniego formatu / nie jest przygotowane api pod niego, to dostaniesz i tak Internal Server Error.

src:
https://flask.palletsprojects.com/en/2.2.x/errorhandling/